### ISSUE TITLE
Why not delete this project....

### ISSUE THREAD

**Issue by gehidore** [helmuthdu/aui#247]
If it's not maintained why keep it around to continually screw up newbies?

**Comment by djfj74** [comment]
Right? It still works, but I will fork it and make it better.﻿

**Comment by Bucsi** [comment]
I completely disagree with @gehidore. It works most of the time, we should not delete it.

**Comment by peileppe** [comment]
Tested both scripts  recently with ArchLinux Current Release: 2015.08.01 / Included Kernel: 4.1.3 / ISO Size: 657.0 MB -- it worked somehow but required a little tweak for the partitioning (adding a Bios Boot partition of 2MiB and relaunching grub-install)

**Comment by tombh** [comment]
It's confusing that the `git blame` date for the README's "Project no longer maintained" title is Feb 2014. But there have been many commits since then!

What's going on!?

**Comment by Bucsi** [comment]
helmuthdu is not developing it anymore. But if someone commits a fix or
something, and it works, he patches it into the original.

2015-09-01 15:24 GMT+02:00 Thomas Buckley-Houston <notifications@github.com>
:

**Comment by tombh** [comment]
Great. But that's not what I'd normally describe as, "no longer maintained". Has there been any discussion of transferring ownership?

What are the odds of having a PR accepted that changed "Project no longer maintained" to "I no longer actively develop this, but will accept PRs"?

**Comment by Bucsi** [comment]
Well, you could create that PR and see what happens ;)
2015. szept. 1. 16:25 ezt írta ("Thomas Buckley-Houston" <
notifications@github.com>):

**Comment by julianxhokaxhiu** [comment]
This project actually works and the author just committed some hours ago. So please close this issue, which is definitely not one.

**Comment by jmayday** [comment]
@gehidore as you can read in referenced ticket, project status is rather 'mature' than 'abandoned/buggy'. If you want to be helpful pls provide list of (better) alternatives at all.
